src/bio2bel/sources/rhea.py

```diff
@@ -0,0 +1,132 @@
+# -*- coding: utf-8 -*-
+
+"""Convert Rhea to BEL."""
+
+import gzip
+import logging
+
+import rdflib
+
+import pybel
+import pybel.dsl as dsl
+from ..utils import ensure_path
```

Comment:
try
```python
from bio2bel.constants import BIO2BEL_MODULE

rdflib_graph = BIO2BEL_MODULE.ensure_rdf('rhea', url=...)
```

Reply:
you'll have to update from master first

Reply:
How do I update the PR from master?

Reply:
Two guides you could follow:
1. https://docs.github.com/en/github/collaborating-with-issues-and-pull-requests/syncing-a-fork
2. https://gist.github.com/CristinaSolana/1885435

I'm going to add you to the repo so next time you don't need a fork :)

Reply:
Should the `rdflib_graph` assignment be inside the `get_bel()` function, or should it be assigned at the top of the file?

Reply:
inside the function. since this is quite expensive and slow, it's better to wait for the user to ask for it

Reply:
Sounds good! I merged in the upstream branch, and I see changes to `BIO2BEL_MODULE` under `constants.py`, but I can't access `ensure_rdf()` for some reason.

When I try to run`g = BIO2BEL_MODULE.ensure_rdf('rhea', url=URL)`, I get the following error: `AttributeError: 'Module' object has no attribute 'ensure_rdf'`.

Reply:
hmm. Try doing `pip install --upgrade pystow` -> I just added this functionality before the weekend

Reply:
That did the trick!